### PR TITLE
[revamp] : announcement UI

### LINKED ISSUE #3525: [revamp] : announcement header ui
**Describe the bug**
The current UI for the announcement is consuming the space. 
New UI:
| Screenshot from 2023-01-25 21-56-28 |
| - |
| New UI |

**Device Information [optional]:**

**Additional context**
This issue is assigned to the core-team as to merge it ASAP.

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by vedant-jain03**
Fixes #3525 

### Screenshots of the changes (If any) -

| Screenshot from 2023-01-25 21-56-28 |
|-|
|New UI|

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -71,4 +72,8 @@ def store_user_location!
     def after_sign_in_path_for(resource_or_scope)
       stored_location_for(resource_or_scope) || super
     end
+
+    def set_announcement
```
Comment (tachyons) [inline app/controllers/application_controller.rb:76]: Use current_announcement helper, no need to create additional methods

**Comment by vedant-jain03** [comment]
cc @tachyons

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-expand-lg bg-white">
+<nav class="<%= show_announcement %>">
```
Comment (tachyons) [inline app/views/layouts/_header.html.erb:1]: You can simply keep existing css classes, and add navbar class conditionally, using a helper makes this code harder to understand

**Review by tachyons — CHANGES_REQUESTED** [review]

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-expand-lg bg-white">
+<nav class="<%= current_announcement ? 'navbar-announcement' : '' %> navbar navbar-expand-lg bg-white">
```
Comment (tachyons) [inline app/views/layouts/_header.html.erb:1]:
```suggestion
<nav class="<%= 'navbar-announcement'  if current_announcement %> navbar navbar-expand-lg bg-white">
```

**Review by tachyons — APPROVED** [review]

**Pull request merged**